components/brave_extension/extension/brave_extension/background/humanWeb.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+declare const CLIQZ: any // this is a global injected by humanweb
+const HUMAN_WEB_PREF_KEY = 'brave.human_web_enabled'
+
+const toggleHumanWeb = (enabled: boolean) => {
+  const cliqzPrefs = CLIQZ.app.prefs
+  if (enabled) {
+    cliqzPrefs.set('modules.human-web.enabled', true)
+    cliqzPrefs.set('modules.hpnv2.enabled', true)
+  } else {
+    cliqzPrefs.set('modules.human-web.enabled', false)
+    cliqzPrefs.set('modules.hpnv2.enabled', false)
```

Comment:
IIUC there are some other modules - are these disabled by default? https://github.com/brave/humanweb/blob/main/configs/humanweb.js#L68

just want to make sure the extension doesn't do anything if the pref is disabled

Reply:
I think we would have more control if we include the humanweb code as a module. There is an example of how this could be done here: https://github.com/brave/humanweb/blob/main/brave-browser-integration-example.js

In the example the behavior is similar to here (only hpnv2 and human-web are enabled/disabled) but we could enable/disable the "app" itself, which would results in all modules being either enabled or disabled. We will have to move to this approach eventually so we could already do it now.

Reply:
we import the extension as an es module now